datafusion/tests/user_defined_plan.rs

```diff
@@ -109,6 +109,22 @@ async fn setup_table(mut ctx: ExecutionContext) -> Result<ExecutionContext> {
     Ok(ctx)
 }
 
+async fn setup_table_without_columns(
```

Comment:
i think without schema would have been more accurate here. same for the other tests.

Reply:
good idea! fixed